README.md

```diff
@@ -89,7 +89,7 @@ For more advanced users, or those who need to integrate CKEditor 5 with their ow
 
 CKEditor 5 builds allow you to quickly and easily initialize one of the many types of editors in your application. At the same time, CKEditor 5 is also a framework for creating custom-made rich text editing solutions.
 
-To find out how to start building your own editor from scratch go to the [CKEditor 5 Framework overview](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/overview.html) section of CKEditor 5 documentation.
```

Comment:
This address is actually redirected to https://ckeditor.com/docs/ckeditor5/latest/framework/index.html, which means the new version no longer works. We need to add a new redirect or change this to the actual link.

Reply:
But, isn't `framework/index.html` a correct link now (and used to be anyway)?

Reply:
Right, now it works, so let's discard this comment.